Util/TimeSamplingInfo.m

```diff
@@ -0,0 +1,144 @@
+function [info] = TimeSamplingInfo(t, prec),
```

Comment:
>I'm lost with the first commit. This is way too hard to review. It doesn't just look like indenting. Some blocks of code seem to have moved and may be modified. Why indenting some files, while the rest of the functions in this repo aren't indented. It doesn't respect the coding style, or at least this is not consistent.

As I said, the commit would be light to review if you use "diff -w". The problem here is that `github diff` got confused about the indentation and cannot resolve it properly (and you can't change the `diff` flags in github).

Yes, I'm linting everything that will be changed in the PR, but I'm keeping the linting on a single commit.  IMO, what I was doing above is the most sensible in the sense that linting is done in a reduced set, with tests, and enclosed to the files affected by that PR. I don't think a "one PR with all linting in the code base" would be easily "reviewable". Worse, it would not be "testable", since code coverage is low. I don't trust a linter to run it without testing the actual code.

I will, from now onwards, follow the current "linting standard"